client/src/components/NavBar/NavBar.tsx

```diff
@@ -110,7 +113,6 @@ export function Navbar(props: any) {
         <Link to={"/single-result/ChIJn58N1B9gUocRpAXOXPbFcOo"}>
           Single Result
         </Link>
-      </details>
```

Comment:
I think the details should be kept. Why do you want to remove?

Reply:
Answer is in PR 140